frontend/src/app/features/in-app-notifications/entry/actors-line/in-app-notification-actors-line.component.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  OnInit,
+} from '@angular/core';
+import { INotification } from 'core-app/core/state/in-app-notifications/in-app-notification.model';
+import { PrincipalLike } from 'core-app/shared/components/principal/principal-types';
+import {
+  Observable,
+  timer,
+} from 'rxjs';
+import {
+  distinctUntilChanged,
+  map,
+} from 'rxjs/operators';
+import { I18nService } from 'core-app/core/i18n/i18n.service';
+import { TimezoneService } from 'core-app/core/datetime/timezone.service';
+import { DeviceService } from 'core-app/core/browser/device.service';
+
+@Component({
+  selector: 'op-in-app-notification-actors-line',
+  templateUrl: './in-app-notification-actors-line.component.html',
+  styleUrls: ['./in-app-notification-actors-line.component.sass'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class InAppNotificationActorsLineComponent implements OnInit {
+  @Input() aggregatedNotifications:INotification[];
+
+  @Input() notification:INotification;
+
+  // The actor, if any
+  actors:PrincipalLike[] = [];
+
+  // Fixed notification time
+  fixedTime:string;
+
+  // Format relative elapsed time (n seconds/minutes/hours ago)
+  // at an interval for auto updating
+  relativeTime$:Observable<string>;
+
+  text = {
+    and: this.I18n.t('js.notifications.center.label_actor_and'),
+    and_other_singular: this.I18n.t('js.notifications.center.and_more_users.one'),
+    and_other_plural: (count:number):string => this.I18n.t('js.notifications.center.and_more_users.other',
+      { count }),
+    loading: this.I18n.t('js.ajax.loading'),
+    placeholder: this.I18n.t('js.placeholders.default'),
+    mark_as_read: this.I18n.t('js.notifications.center.mark_as_read'),
+    updated_by_at: (age:string):string => this.I18n.t('js.notifications.center.text_update_date',
+      { date: age }),
+  };
+
+  constructor(
+    private I18n:I18nService,
+    private timezoneService:TimezoneService,
+    private deviceService:DeviceService,
+  ) { }
+
+  ngOnInit():void {
+    this.buildActors();
+    this.buildTime();
+  }
+
+  isMobile():boolean {
+    return this.deviceService.isMobile;
```

Comment:
I don't think you need the overhead of a function call here. I'd just check for `deviceService.isMobile` in the template directly